packages/fabric/src/index.ts

```diff
@@ -0,0 +1,169 @@
+import { Restroom } from "@restroom-mw/core";
+import { ObjectLiteral } from "@restroom-mw/core/src/types";
+import * as crypto from 'crypto';
+// How to include this?
+// import { combineDataKeys } from "@restroom-mw/sawroom/src/lib";
+import { Gateway, connect, Contract, Identity, Signer, signers,
+         Network }
+   from '@hyperledger/fabric-gateway';
+import { Client, credentials } from '@grpc/grpc-js';
+import axios from "axios";
+import { NextFunction, Request, Response } from "express";
+import {
+  ADDRESS,
+  CONNECT,
+  CHANNEL,
+  CONTRACT,
+  QUERY,
+  SUBMIT
+} from "./actions";
+import { TextDecoder } from 'util';
+
+let fabricAddress: string = null;
+let client: Client = null;
+let tlsCertificate;
+let input: ObjectLiteral = {};
+let gateway: Gateway = null;
+let network: Network = null;
+let contract: Contract = null;
+
+const utf8Decoder = new TextDecoder();
```

Comment:
Since this is a single instance that you use, you can already create the instance in the `util` and import it directly as `utf8Decoder`, how do you see that? Sorry maybe getting to stickler ;p